### ISSUE TITLE
Maybe there's a conflict with VSCode's sticky-scroll

### ISSUE THREAD

**Issue by ItsUncharted** [longfish801/regexpOutline#3]
The extension is very helpful in most cases, but when I write a file with only variables, it does show the outline correctly, but vscode's sticky-scroll effect seems to be lost. 

When the variables have multi multi lines, due to the lack of sticky-scroll, it's quite hard to recognize which one is being viewed and edited (meanwhile functions and methods' outline and sticky-scoll both work well), and I cannot find a way to get the effect back unless I uninstall the extension.

I've tried different similar extensions, it all seemed to have this problem, maybe that's because vsc just dont support sticky-scoll in such way at this point :( ...

**Comment by longfish801** [comment]
Sorry, RegExp Outline does not support multi-line headings.
I wrote it [Features of README](https://github.com/longfish801/regexpOutline?tab=readme-ov-file#features) as "Single-line headings are assumed. Complex syntax is not supported."
Sticky-scroll is not the cause.

**Comment by ItsUncharted** [comment]
Actually the outline worked normally when sticky-scroll was enabled, and I did use single-line name for variables.
What I mean is when I used this extension, VSCode's sticky-scroll effect just disappeared, the extension itself was all good.

I've tried different programming languages, this weird problem only occurred in some particular cases. So it should be related to vsc or other extensions, many thanks to such a helpful extension.
